### PR TITLE
docs: add instructions for installing on Arch Linux

### PR DESCRIPTION
**Pull request by orhun**
addresses https://github.com/kyu08/fzf-make/pull/56#issuecomment-1684944633

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ brew update
 brew upgrade fzf-make
 ```
 
+## Arch Linux
```
Comment (kyu08) [inline README.md:37]: I just realized that it is easier to understand in the format like `## ${name_of_package_manager} (${OS})`. (sorry for pointing this out after you added it)

So could you to modify to this.

```suggestion
## paru (Arch Linux)
```

If `paru` is not name of  package manager, please fix it. 🙇‍♂️

**Review by kyu08 — COMMENTED** [review]
Thanks for helping me add documentation!

I would like to see one correction. 🙇‍♂️

File: README.md (hunk 1 above)
Comment (orhun) [reply README.md:37]: AUR helpers are not [officially supported](https://wiki.archlinux.org/title/AUR_helpers) so I would say let's not suggest it in the title and keep it in the example.

Comment (kyu08) [reply README.md:37]: I see.
So as you said, it's fine as it is. 👍

**Review by kyu08 — APPROVED** [review]
Thanks!

**Pull request merged**